system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -173,23 +173,26 @@
   "Create an expected concept given a service, its concept-id, a revision-id,
   and a user-id."
   [variable concept-id revision-id user-id]
-  {:concept-type :variable
-   :native-id (string/lower-case (:variable-name variable))
-   :provider-id "CMR"
-   :format mt/edn
-   :metadata (pr-str variable)
-   :user-id user-id
-   :deleted false
-   :concept-id concept-id
-   :revision-id revision-id})
+  (let [native-id (string/lower-case (:Name variable))]
+    {:concept-type :variable
+     :native-id native-id
+     :provider-id "CMR"
+     :format mt/edn
+     :metadata (pr-str (assoc (util/kebab-case-data variable)
```

Comment:
I think we can skip the checking on metadata in edn format here as we are changing it to JSON format and it would be much easier to compare when the metadata doesn't change.